### PR TITLE
Updated Indentation

### LINKED ISSUE #1815: #1425  algorithm for left rotation is available but algorithm for right rotation is missing in C,Java,C++,Python
The algorithm for left rotation thats `1234 ---> 4123` is available but the algorithm for right rotation is missing example `1234 --> 2341 --->3412 ---> 4123` is not available I would not only like to add the algorithm and I  would also like to add a Readme.Md for the same.Please assign me this issue as a part of GSSoC 2020 @SakshiShreya and @jainaman224

### PR DESCRIPTION
**Pull request by CSElonewolf**
Closes #1815 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Right_Rotation_Of_Number
- Updated The indentation

#### Languages Used:
- Java

#### Files Added:
-  Added Right_Rotation_Of_Number

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+ class Right_Rotation_Of_Number
+ {
+ public static void main(String args[])
+          {
```
Comment (jainaman224) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Use 4 space indentation.

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+ class Right_Rotation_Of_Number
+ {
+ public static void main(String args[])
```
Comment (jainaman224) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Inconsistent indentation.

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          {
+		   Scanner obj=new Scanner(System.in);
+		   int num=obj.nextInt();
+		   Right_Rotation(num);
+	  }
+	 //Right_Rotation method to rotate the number
```
Comment (jainaman224) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Add spaces after `//`.

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+ class Right_Rotation_Of_Number
+ {
+ public static void main(String args[])
+          {
+		   Scanner obj=new Scanner(System.in);
+		   int num=obj.nextInt();
```
Comment (jainaman224) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Add spaces around operators like `==`. `=`, `+=`, `+`.

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+ class Right_Rotation_Of_Number
+    {
```
Comment (somya-kapoor) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Please correct the indentation of {

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+ public static void main(String args[])
+          {
+		   Scanner obj = new Scanner(System.in);
+		   int num = obj.nextInt();
+		   Right_Rotation(num);
+	  }
+	 // Right_Rotation method to rotate the number
+	public static void Right_Rotation(int num) 
+	  {
```
Comment (somya-kapoor) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Please correct the indentation of {

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+		// Convert the number to string
+		String number = String.valueOf(num);
+		// Find the length of the number to run the loop
+		int len = number.length();
+		// Loop runs 1 less the length of the number.
+		for(int i = 1;i < len;i++)
```
Comment (somya-kapoor) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Please add space after ;

File: Rotation_Of_Numbers/Right_Rotation_Of_Number.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+		    {
+			// Main rule to rotate the number
+			// eg:-old number 12345 ---> how the number splits(2345 + 1) 
+			// new number = 23451 
+			number=number.substring(1,len)+number.substring(0,1);
```
Comment (somya-kapoor) [inline Rotation_Of_Numbers/Right_Rotation_Of_Number.java]: Please add space around the operators.

**Comment by somya-kapoor** [comment]
@CSElonewolf Please use 4 space indentation

**Comment by somya-kapoor** [comment]
@CSElonewolf Are you working on it?

**Comment by somya-kapoor** [comment]
@CSElonewolf Are you working on it?

**Pull request closed without merging**